test/conformance/api/v1/service_validation_test.go

```diff
@@ -44,14 +45,39 @@ func TestServiceAccountValidation(t *testing.T) {
 	test.CleanupOnInterrupt(func() { test.TearDown(clients, names) })
 
 	t.Logf("Creating a new Service %s", names.Service)
-	service := v1test.Service(names, WithServiceAccountName(invalidServiceAccountName))
-	v1test.LogResourceObject(t, v1test.ResourceObjects{Service: service})
+	_, err := v1test.CreateService(t, clients, names, WithServiceAccountName(invalidServiceAccountName))
 
-	_, err := clients.ServingClient.Services.Create(service)
 	if err == nil {
 		t.Fatal("Expected Service creation to fail")
 	}
 	if got, want := err.Error(), "serviceAccountName: spec.template.spec."+invalidServiceAccountName; !strings.Contains(got, want) {
 		t.Errorf("Error = %q, want to contain = %q", got, want)
 	}
 }
+
+func TestServiceValidationWithInvalidPodSpec(t *testing.T) {
```

Comment:
I think we should move this to the [e2e test package](https://github.com/knative/serving/tree/master/test/e2e) for now.

Few reasons why it can't just be part of conformance as is:
1) This change would need to be reflected in other API versions (ie. alpha & beta)
2) Needs buy-in from vendors/community - since this would be adding additional requirements

Reply:
Totally agreed. Moved.